Voter/ReportVoterInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Staffim\RollbarBundle\Voter;
+
+interface ReportVoterInterface
+{
+    public function vote($exception);
```

Comment:
I wasn't sure about typeHinting as I'm not very sure about the error to exception conversion.

But if we stick with this strategy, there is some typehint to add…